socorro/unittest/lib/test_javautil.py

```diff
@@ -131,13 +149,6 @@ def test_parse_caused_by():
         "",
         # Line without a tab in STACK stage
         ("Exception: msg\n" "\tat org.File.function(File.java:100)\n" "badline"),
-        # Line without a tab in ADDITIONAL stage
-        (
-            "Exception: msg\n"
-            "\tat org.File.function(File.java:100)\n"
-            "\tSuppressed: Exception: msg\n"
-            "badline"
-        ),
```

Comment:
This was a malformed case because `badline` didn't have a tab at the beginning of the line. That's no longer something we consider malformed, so I nixed this test case.